internal/services/access_flow.go

```diff
@@ -0,0 +1,476 @@
+package services
+
+import (
+	"context"
+	"fmt"
+	"github.com/apono-io/apono-sdk-go"
+	"github.com/apono-io/terraform-provider-apono/internal/models"
+	"github.com/apono-io/terraform-provider-apono/internal/utils"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"golang.org/x/exp/slices"
+	"math/big"
+	"strings"
+)
+
+func ConvertToAccessFlowModel(ctx context.Context, aponoClient *apono.APIClient, accessFlow *apono.AccessFlowV1) (*models.AccessFlowModel, diag.Diagnostics) {
+	data := models.AccessFlowModel{}
+	data.ID = types.StringValue(accessFlow.GetId())
+	data.Name = types.StringValue(accessFlow.GetName())
+	data.Active = types.BoolValue(accessFlow.GetActive())
+	data.RevokeAfterInSec = types.NumberValue(big.NewFloat(float64(accessFlow.GetRevokeAfterInSec())))
```

Comment:
in here and other methods in this file
I  think resolving data and then assigning it in one place in the end of the method makes it more readable and easier to see the whole picture
```
a = resolveA()
code
code
b = resolveB(code)
c = ...

data := models.AccessFlowModel{
  A: a,
  B: b,
  C: c
}
```

Reply:
also relevant to other heavy conversion methods in here